/**:
  ros__parameters:
    # global_params
    max_translational_velocity: 0.3
    max_rotational_velocity: 1.0
    direction_linear_if_alone: 0.0
    direction_angular_if_alone: 2.0
    # attraction_pattern
    attraction_max_range: 1.50    #0.33
    attraction_min_range: 0.15    #0.08
    attraction_front_attraction: 0.0
    attraction_threshold: 1
    # disperrsion_pattern
    dispersion_max_range: 0.8
    dispersion_min_range: 0.04          
    dispersion_front_attraction: -1.0
    dispersion_threshold: 1
    # random_pattern
    random_walk_timer_period: 0.2
    random_walk_linear: 0.15
    random_walk_angular: 3.0
    random_walk_rot_interval: 2.0
    random_walk_lin_interval_min: 2.5
    random_walk_lin_interval_max: 4.5
